cosmos/providers/dbt/parser/project.py

```diff
@@ -124,30 +118,28 @@ def __post_init__(self) -> None:
             self.name = snapshot_name
             code = code.split("%}")[1]
             code = code.split("{%")[0]
-
         elif self.type == DbtModelType.DBT_SEED:
             code = None
-
         # get the dependencies
         env = jinja2.Environment()
         ast = env.parse(code)
-
         # iterate over the jinja nodes to extract info
-        for base_node in ast.find_all(jinja2.nodes.Call):
-            if hasattr(base_node.node, "name"):
+        for jinja_call_node in ast.find_all(Call):
+            if hasattr(jinja_call_node.node, "name"):
                 # check we have a ref - this indicates a dependency
-                if base_node.node.name == "ref" or base_node.node.name == "source":
+                # TODO Do we care about source references? (unless we add source freshness) also a source takes 2 args.
```

Comment:
yeah, we probably need to do a full spike on sources - there are lots of ways to handle them. I think we have a few issues opened. I'd say let's keep the behavior as-is for now and do a deeper dive into sources as a follow-on.